validate_hif.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+from collections import defaultdict
+
+# network parameters
+filename = sys.argv[1]
+
+with open(filename) as file:
+    # load JSON file
+    data = json.loads(file.read())
```

Comment:
In case of an exception the code fails.
The *library* version should gracefully throw error.
The *executor* version should print to `stderr` when the `--silent` is not set and yield exit status equal to 1